tests/qchem/openfermion_pyscf_tests/test_convert_openfermion.py

```diff
@@ -81,31 +80,17 @@ def test_tol_qubit(self):
             "Z1", complex(1.3, 1e-8)
         )
 
-        pl_op = _from_openfermion_qubit(q_op, tol=1e-6)
+        pl_op = qml.from_openfermion(q_op, tol=1e-6)
         assert not np.any(pl_op.coeffs.imag)
 
-        pl_op = _from_openfermion_qubit(q_op, tol=1e-10)
+        pl_op = qml.from_openfermion(q_op, tol=1e-10)
         assert np.any(pl_op.coeffs.imag)
 
     def test_sum_qubit(self):
-        """Test that the from_openfermion_qubit method yields a :class:`~.Sum` object if requested."""
+        """Test that from_openfermion yields a :class:`~.Sum` object."""
         q_op = openfermion.QubitOperator("X0 X1", 0.25) + openfermion.QubitOperator("Z1 Z0", 0.75)
 
-        assert isinstance(_from_openfermion_qubit(q_op), qml.ops.LinearCombination)
-        assert isinstance(
-            _from_openfermion_qubit(q_op, format="LinearCombination"), qml.ops.LinearCombination
-        )
-        assert isinstance(_from_openfermion_qubit(q_op, format="Sum"), qml.ops.Sum)
-
-    def test_invalid_format_qubit(self):
```

Comment:
Is there a reason we are removing this test?

Reply:
We do not have format anymore. Just returning a LinearCombination.